grr/server/grr_response_server/gui/ui/lib/api/api_interfaces.ts

```diff
@@ -163,6 +163,26 @@ export declare interface ApiClient {
   readonly lastBootedAt?: string;
   readonly lastClock?: string;
   readonly labels?: ReadonlyArray<ApiClientLabel>;
+
+  readonly age?: string;
+}
+
+/**
+ * ApiGetClientVersionsResult proto mapping.
+ */
+export declare interface ApiGetClientVersionsResult {
+  readonly items: ReadonlyArray<ApiClient>;
+}
+
+/**
+ * ApiGetClientVersionsArgs proto mapping.
+ */
+export declare interface ApiGetClientVersionsArgs {
+  readonly clientId: string;
```

Comment:
Missing `?`.

Reply:
I decided to not use `?` as the `clientId` is a must have argument in order to get the client versions. Without it, we wouldn't be able to fetch the client versions, so it would be an incorrect use.